styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.md

```diff
@@ -0,0 +1,7 @@
+---
+el: '.card-course'
+title: 'Course Card'
+---
+The Course Card is a variant of the default card style and consists of a required image, course duration, required title, course credits, course date, and optional description, as well a required block level link.  Text fields in this component should have sensible character limits set to enforce a rough maximum height for the cards.
```

Comment:
Can we give recommendations as to what those sensible character limits should be?